hacking/aws_config/testing_policies/compute-policy.json

```diff
@@ -245,6 +245,19 @@
             "Resource": [
               "arn:aws:states:*"
             ]
+        },
+        {
+            "Sid": "AllowLightsail",
```

Comment:
If you want to also add a PR to this repo and file https://github.com/mattclay/aws-terminator/blob/master/aws/policy/compute.yaml with these, it's where we deploy the permissions for CI. The hacking/ directory policies make running the tests in contributors' own accounts more convenient but the permissions aren't all strict enough to actually use for CI.

Reply:
Hi @s-hertel. I took a crack at it, but I am not very sure about the change. Please let me know if this works -
https://github.com/mattclay/aws-terminator/pull/68